<?xml version="1.0"?>
<launch>
  <arg name="robot_namespace" default="scout"/>
<!--  <group ns="$(arg robot_namespace)"> -->
    <node name="sub_goal_pid" pkg="simple_pid_controller" type="sub_goal_pid" respawn="false" clear_params="true" output="screen">
      <rosparam file="$(find simple_pid_controller)/config/corsair.yaml" />
    </node>
  <!--</group> -->

</launch>
